config/install/core.extension.yml

```diff
@@ -0,0 +1,96 @@
+module:
+  admin_toolbar: 0
+  admin_toolbar_links_access_filter: 0
+  admin_toolbar_tools: 0
+  allowed_formats: 0
+  anchor_link: 0
+  big_pipe: 0
```

Comment:
Pipe? Really?

Reply:
why not? better performance

Reply:
How does this play with varnish?

https://www.drupal.org/docs/8/core/modules/big-pipe/bigpipe-environment-requirements#varnish
https://docs.acquia.com/acquia-cloud/performance/bigpipe/

Looks like Acquia recommends the https://www.drupal.org/project/big_pipe_sessionless module to help with anon traffic. This is likely something we will want to keep "hits" down.

Reply:
Hmmm..
>>> So, when using Varnish: the first request will be streamed by BigPipe, and not cached in Varnish. The second request will be a Page Cache hit, not a Varnish hit. The third request and all later requests will be Varnish hits.

Reply:
Lets pull big pipe out and put in a new ticket to explore this. We can't afford to have several hits per page on our factory subscription.

Reply:
I've been using it on swshumsci and havent seen a spike in page requests

Reply:
although i do need to review the logs more

Reply:
we can investigate later.

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-329